server/monitor/src/main/java/org/apache/accumulo/monitor/Monitor.java

```diff
@@ -642,7 +643,7 @@ private void getMonitorLock() throws KeeperException, InterruptedException {
       UtilWaitThread.sleep(getContext().getConfiguration().getTimeInMillis(Property.MONITOR_LOCK_CHECK_INTERVAL));
     }
 
-    log.info("Got Monitor lock.");
+    log.info("Acquired Monitor Lock " + monitorLock.getLockPath());
```

Comment:
Any particular reason this would be better than what I used? I am not familiar with the slf4j format and I did it the way I did since some debug statements where written like that.